server/store/store.go

```diff
@@ -15,15 +14,15 @@ type pluginStore struct {
 }
 
 // NewPluginStore creates Store object from plugin.API
-func NewPluginStore(api plugin.API) Store {
+func NewPluginStore(api store.API) Store {
 	return &pluginStore{
 		userStore: store.NewPluginStore(api),
 	}
 }
 
-// NewWithStores creates Store object from stores
-func NewWithStores(userStore store.KVStore) Store {
-	return &pluginStore{
-		userStore: userStore,
-	}
-}
+//// NewWithStores creates Store object from stores
```

Comment:
Don't we need this function anymore?

Reply:
Since the Store.API interface is mockable, there is no need for a custom initializer any more

Reply:
It's better to remove this code than